### DISCUSSION TITLE
Is there a way to add general use-case spellchecker

### DISCUSSION THREAD

**Discussion by Na3aga** [zed-industries/zed#7966] [Feature Requests]
I am trying to use Zed instead of VScode for Rust primarily. But I am really missing a spellchecker though, it just lets me focus more on other stuff instead of naming correctness I use, so I won't find any wrong letters in some variable or field names later. Just like the Grammarly autocompletion that I use even to write this feature request ))) 

Is there solution for that?

**Comment by SomeoneToIgnore** [comment]
We miss that too and even have the issue for that: https://github.com/zed-industries/zed/issues/5305

Unfortunately, there's no support for any kind of spellcheck in Zed, and it would require a design and some Rust code to support any.
